src/expr/proof_rule.h

```diff
@@ -507,6 +507,69 @@ enum class PfRule : uint32_t
   // sigma maps x1 ... xn to t1 ... tn.
   INSTANTIATE,
 
+  // Note: an ArithLiteral is a term of the form (>< poly const)
+  // where
+  //   >< is >=, >, ==, <, <=, or not(== ...).
+  //   poly is a polynomial
+  //   const is a rational constant
+
```

Comment:
Can you make sure this is formatted the same as above (e.g. with `======`).

Reply:
Sure. Done in  8c5438338.